src/main/java/io/reactiverse/awssdk/converters/VertxAsyncResponseTransformer.java

```diff
@@ -0,0 +1,51 @@
+package io.reactiverse.awssdk.converters;
+
+import io.reactiverse.awssdk.reactivestreams.WriteStreamSubscriber;
+import io.vertx.core.Handler;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.streams.WriteStream;
+import software.amazon.awssdk.core.async.AsyncResponseTransformer;
+import software.amazon.awssdk.core.async.SdkPublisher;
+
+import java.nio.ByteBuffer;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+
+public class VertxAsyncResponseTransformer<ResponseT> implements AsyncResponseTransformer<ResponseT, WriteStream<Buffer>> {
+
+    private volatile CompletableFuture<WriteStream<Buffer>> cf;
+    private volatile ResponseT response;
+    private volatile WriteStream<Buffer> writeStream;
+    private volatile Optional<Handler<ResponseT>> responseHandler;
+
+    public VertxAsyncResponseTransformer(WriteStream<Buffer> ws) {
+        this.writeStream = ws;
```

Comment:
I think we should initialise the responseHandler in the constructor, otherwise using the ResponseTransformer without invoking `setResponseHandler` will result in a null pointer exception.
`this.responseHandler = Optional.empty();`